### ISSUE TITLE
Solved plugin problems?

### ISSUE THREAD

**Issue by jeremycherfas** [getgrav/grav-plugin-comments#77]
I think I may have solved the problem of the plugin not working.

Line 192 originally read `$post = isset($_POST['data']) ? $_POST['data'] : [];`

However, `$_POST` as received from the form did not contain any element `data`. I replaced that line with the admittedly simpler
````
If (isset($_POST)){
  $post = $_POST;
}
````

The plugin now works in my installation. I didn't actually fork the plugin, and so am not sure how to file a PR.

I also reversed `email` and `addComment` in the process section of `comments.yaml` to avoid the problem of receiving an email even though the comment was not saved.

This fix works even without `language` being set.

**Comment by Romarain** [comment]
@jeremycherfas  Just tried all what you said, and it doesn't work for me. Nothing is saved in the `/data` folder.

**Comment by jeremycherfas** [comment]
I'm afraid I cannot help you. But let me test the comment function on my own site. I get so few comments that I can never be sure that it still works after upgrading Grav.

**Comment by Romarain** [comment]
@jeremycherfas Yes, I talked about that. But you know, for third-party plugins, people tend to ignore requests that seem more a consequence of our own config that of the plugin itself.

**Comment by jeremycherfas** [comment]
Not sure what you mean about 3rd party. This plugin is made by the Grav team.
